AMD GPU Driver 30.10.2 initial changes added (#41)
ROCm/amdgpu-docs: docs/documentation/change-logs/30.10.2.md

# AMD GPU Driver (amdgpu) 30.10.2 release notes

> The release notes provide a summary of notable changes since the previous AMD GPU Driver release.

Reviewer: Shouldn't we indicate the previous version number. 
Perhaps in parentheses at the end of the sentence.

## Release highlights

The following are notable new features and improvements in AMD GPU Driver 30.10.2.

### Operating system and hardware support changes

The AMD GPU Driver 30.10.2 introduces support for the following operating systems:

* RHEL 10.0

* Debian 13

For the compatibility between AMD GPU Driver, ROCm, GPUs, and OS, see the [Compatibility matrix](../../compatibility/compatibility-matrix.rst).

### GPU resiliency

> Multimedia Engine Reset for finer-grain GPU Resiliency is now supported in the AMD GPU Driver 30.10.2 for AMD Instinct MI300X GPUs. It allows recovery from VCN or JPEG related faults without requiring a full GPU reset, improving system stability and fault tolerance.
> To support this feature, the AMD Instinct MI300X GPU requires PLDM bundle 01.25.05.00 (or later) firmware and AMD GPU Driver (amdgpu) 30.10.2.

Reviewer: ```suggestion
Multimedia Engine Reset for finer-grain GPU Resiliency is now supported in the AMD GPU Driver 30.10.2 for AMD Instinct MI300X GPUs. It allows recovery from VCN or JPEG-related faults without requiring a full GPU reset, thereby improving system stability and fault tolerance.
To support this feature, the AMD Instinct MI300X GPU requires PLDM bundle 01.25.05.00 (or later) firmware and AMD GPU Driver (amdgpu) 30.10.2.
```

### PCIe Error Recovery

Downstream Port Containment (DPC) and Advanced Error Reporting (AER) are PCIe mechanisms that work together to detect and recover from hardware errors. Enabling DPC for AER allows the system to isolate faulty PCIe devices and recover gracefully without crashing. This feature is supported on the AMD Instinct MI300X, MI300A, and MI325X GPUs.

## Known issues

> ROCm 7.0.2 has a known multimedia issue on AMD Instinct MI300A GPU when paired with BKC 26. If used, subsequent multimedia jobs might become unstable after the multimedia engine reset was performed to recover a VCN related fault. BKC 25 or older does not have this issue. This issue will be fixed in an upcoming release.

Reviewer: ```suggestion
ROCm 7.0.2 has a known multimedia issue with AMD Instinct MI300A GPU when paired with BKC 26. If used, subsequent multimedia jobs might become unstable after the multimedia engine is reset to recover a VCN-related fault. BKC 25 or older does not have this issue. This issue will be fixed in an upcoming release.
```
